assets/css/order-styles.css

```diff
@@ -21,8 +23,9 @@
 	vertical-align: middle;
 }
 
-.type-shop_order .column-wc_actions a.button.wpo_wcpdf img,
-.type-shop_order .column-order_actions a.button.wpo_wcpdf img /* legacy: 3.2 & older */ {
+.type-shop_order .column-order_actions a.button.wpo_wcpdf img, /* legacy: 3.2 & older */
+.type-shop_order .column-wc_actions a.button.wpo_wcpdf img, /* pre WC 7.1 */
+.woocommerce_page_wc-orders .column-wc_actions a.button.wpo_wcpdf img {
 	width: 16px;
 }
```

Comment:
There is still a difference between the buttons pre-HPOS and after. The icons are not correctly centered.

Before:
<img width="157" alt="Schermafbeelding 2022-12-02 om 16 45 32" src="https://user-images.githubusercontent.com/19350150/205331499-d5eb4b7c-ee3e-44d7-a713-ee22ecbba5dc.png">

After:
<img width="153" alt="Schermafbeelding 2022-12-02 om 16 45 24" src="https://user-images.githubusercontent.com/19350150/205331523-3d49b050-482d-4f39-ab80-abd9400ec59b.png">